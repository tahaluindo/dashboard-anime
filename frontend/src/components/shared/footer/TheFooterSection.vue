<template>
  <section class="flex flex-col w-full items-start gap-3">
    <slot />
  </section>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="scss">
</style>