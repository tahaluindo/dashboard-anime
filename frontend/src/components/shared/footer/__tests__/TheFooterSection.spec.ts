import { mount } from "@vue/test-utils";
import TheFooterSection from "../TheFooterSection.vue";

describe("TheFooterSection component test", () => {
  it("should render the default component", () => {
    const wrapper = mount(TheFooterSection, {
      slots: {
        default: "<div> SLOT </div>",
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
