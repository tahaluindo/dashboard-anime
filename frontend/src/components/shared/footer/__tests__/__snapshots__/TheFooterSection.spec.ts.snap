// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooterSection component test should render the default component 1`] = `
"<section class=\\"flex flex-col w-full items-start gap-3\\">
  <div> SLOT </div>
</section>"
`;
